go/noise/laplace_noise_test.go

```diff
@@ -172,6 +172,525 @@ func TestDeltaForThresholdLaplace(t *testing.T) {
 		}
 	}
 }
+func TestInverseCDFLaplace(t *testing.T) {
+	for _, tc := range []struct {
+		desc                     string
+		mean, lambda, prob, want float64
+	}{
+		// 2 high-precision calculated random tests.
```

Comment:
it is not quite clear what you mean by high-precision (and I am also not sure that it is relevant)

also, I believe you mean "arbitrary" rather than "random"

so i thinkt it suffices to say somethinglike:
"Two arbitrary test cases."

Also, I don't think it is necessary to have a comment at all when the description says essentially the same thing